packages/pipeline-editor/src/SubmitNotebook.ts

```diff
@@ -1,313 +0,0 @@
- */
-export class SubmitNotebook extends Widget
-  implements Dialog.IBodyWidget<ISubmitNotebookConfiguration> {
-  _envVars: string[];
-  _runtimes: any;
-  _runtimeImages: IDictionary<string>;
-
-  constructor(
-    envVars: string[],
-    runtimes: any,
-    runtimeImages: IDictionary<string>
-  ) {
-    super();
-
-    this._envVars = envVars;
-    this._runtimes = runtimes;
-    this._runtimeImages = runtimeImages;
-
-    this.node.appendChild(this.renderHtml());
-    (this.node.getElementsByClassName(
-      'elyra-form-runtime-config'
-    )[0] as HTMLSelectElement).value = '';
-    (this.node.getElementsByClassName(
-      'elyra-form-framework'
-    )[0] as HTMLSelectElement).value = '';
-  }
-
-  /**
-   * Render the dialog widget used to gather configuration information
-   * required to submit/run the notebook remotely
-   */
-  renderHtml(): HTMLElement {
-    const tr = '<tr>'; //'<tr style="padding: 1px;">';
-    const td = '<td>'; //'<td style="padding: 1px;">';
-    const td_colspan2 = '<td colspan=2>'; //'<td style="padding: 1px;" colspan=2>';
-    const td_colspan3 = '<td colspan=3>'; //'<td style="padding: 1px;" colspan=3>';
-    //var td_colspan4 = '<td colspan=4>'; //'<td style="padding: 1px;" colspan=4>';
-
-    const htmlContent = document.createElement('div');
-    let runtime_options = '';
-    let runtimeImages_options = '';
-
-    for (const key in this._runtimes) {
-      runtime_options =
-        runtime_options +
-        `<option value="${this._runtimes[key]['name']}">${this._runtimes[key]['display_name']}</option>`;
-    }
-
-    for (const image in this._runtimeImages) {
-      runtimeImages_options =
-        runtimeImages_options +
-        `<option value="${image}" >${this._runtimeImages[image]}</option>`;
-    }
-
-    const content =
-      '' +
-      '<table id="table-submit-dialog" class="elyra-table"><tbody>' +
-      tr +
-      td_colspan2 +
-      '<label for="runtime_config">Runtime Config:</label>' +
-      '<br/>' +
-      '<select id="runtime_config" class="elyra-form-runtime-config">' +
-      runtime_options +
-      '</select>' +
-      '</td>' +
-      td_colspan2 +
-      '<label for="framework">Runtime images:</label>' +
-      '<br/>' +
-      '<select id="framework" class="elyra-form-framework">' +
-      runtimeImages_options +
-      '</select>' +
-      '</td>' +
-      '</tr>' +
-      // + tr
-      // + td
-      // +'<label for="cpus">CPUs:</label>'
-      // +'<br/>'
-      // +'<input type="text" id="cpus" name="cpus" placeholder="1" value="1"/>'
-      // +'</td>'
-      //
-      // + td
-      // +'<label for="gpus">GPUs:</label>'
-      // +'<br/>'
-      // +'<input type="text" id="gpus" name="gpus" placeholder="0" value="0"/>'
-      // +'</td>'
-      //
-      // + td
-      // +'<label for="memory">Memory:</label>'
-      // +'<br/>'
-      // +'<input type="text" id="memory" name="memory" placeholder="1Gb" value="1Gb"/>'
-      // +'</td>'
-      // +'</tr>'
```

Comment:
@lresende this PR will remove any trace of these commented out sections. Are you ok with that?